src/RecoveryServices/RecoveryServices.Backup.Helpers/Conversions/RetentionPolicyConversions.cs

```diff
@@ -442,7 +488,41 @@ public static ServiceClientModel.LongTermRetentionPolicy GetServiceClientLongTer
             {
                 serviceClientRetPolicy.YearlySchedule = GetServiceClientLTRYearlySchedule(psRetPolicy.YearlySchedule);
             }
+            return serviceClientRetPolicy;
+        }
+
+        /// <summary>
+        /// Helper function to convert service long term retention policy from ps retention policy.
+        /// </summary>
+        public static ServiceClientModel.LongTermRetentionPolicy GetServiceClientVaultRetentionPolicy(
+            VaultRetentionPolicy psRetPolicy)
+        {
+            if (psRetPolicy == null)
+            {
+                return null;
+            }
+
```

Comment:
@hiaga No, not necessary. Since this is a common helper, I figured it would be best to parse the int for SnapshotRetentionInDays within AzureFilesPsBackupProvider directly from the VaultRetentionPolicy